support/ebpf/python_tracer.ebpf.c

```diff
@@ -253,13 +260,20 @@ static EBPF_INLINE ErrorCode get_PyFrame(const PyProcInfo *pyinfo, void **frame)
       return ERR_PYTHON_BAD_THREAD_STATE_FRAME_ADDR;
     }
 
-    // Get _PyCFrame.current_frame
-    if (bpf_probe_read_user(frame, sizeof(void *), cframe_ptr + pyinfo->PyCFrame_current_frame)) {
-      DEBUG_PRINT(
-        "Failed to read _PyCFrame.current_frame at 0x%lx",
-        (unsigned long)(cframe_ptr + pyinfo->PyCFrame_current_frame));
-      increment_metric(metricID_UnwindPythonErrBadCFrameFrameAddr);
-      return ERR_PYTHON_BAD_CFRAME_CURRENT_FRAME_ADDR;
+    // Get _PyCFrame.current_frame.
+    //
+    // Starting with Python 3.13, the indirection through _PyCFrame is removed.
+    // See commit 006e44f9 in the CPython repo.
+    if (pyinfo->version < 0x30d) {
```

Comment:
Could this just test `PyCFrame_current_frame` to be non-zero instead of version check. That should reduce byte code length a bit by avoiding extra load.

Reply:
We could do that, though I feel that it reduces readability a bit, because it relies on the remote code in user-space to set it to zero after the version check. That being said, with the comment above it should probably still be clear. Happy to adjust, no strong preference in either direction.

Reply:
Fair enough. I just increasingly dislike version comparisons in ebpf. How about we start splitting them to `u8` per-branch booleans?

Reply:
Well. Since there's other locations to change also, I can look at it as a follow up PR and do it on one go then to see net effect. Approving. Thanks!